build/ci/github_actions/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
+set +o pipefail
+
+are_pods_ready (){
+  pods=$@
+  for pod in $pods; do
+    running_containers_count=`echo $pod | awk -F / '{print$1}'`
+    total_containers_count=`echo $pod | awk -F / '{print$2}'`
+    if [ $running_containers_count != $total_containers_count ]; then
+      echo false
+      break
+    fi
+  done
+  echo true
+}
+
+is_kubernetes_cluster_ready (){
+  pods=`kubectl get pods -A | awk '{print$3}' | grep -iv ready`
+  are_all_pods_ready=$(are_pods_ready $pods)
+  if [ $are_all_pods_ready == "true" ]; then
+      echo true
+      break
+  fi
+  
+  echo false
```

Comment:
why is there an `if` and a `break` here?

Reply:
I'm not sure I'm following.
why can't all these lines just be `echo $are_all_pods_ready` instead?

Reply:
done